src/test/java/seedu/address/ui/PersonCardTest.java

```diff
@@ -0,0 +1,34 @@
+package seedu.address.ui;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static seedu.address.testutil.Assert.assertThrows;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import javafx.embed.swing.JFXPanel;
+import seedu.address.testutil.PersonBuilder;
+
+public class PersonCardTest {
+
+    @BeforeAll
+    public static void initJfxRuntime() {
+        new JFXPanel();
```

Comment:
Currently fails github workflow. Try to set the java version to `17.0.11` in the workflow file.